manager/controllers/app/fybrikapplication_controller_unit_test.go

```diff
@@ -297,60 +296,6 @@ func TestDenyOnRead(t *testing.T) {
 	g.Expect(res).To(gomega.BeEquivalentTo(ctrl.Result{}), "Requests another reconcile")
 }
 
-// Tests selection of read-path module
```

Comment:
Why is this test being removed?  How is it related to this PR?

Reply:
Not related, it's just a redundant test... I'll take it out of this PR.